packages/calypso-e2e/src/lib/pages/individual-purchase-page.ts

```diff
@@ -0,0 +1,52 @@
+import { Page } from 'playwright';
+
+const staticSelectors = {
+	renewNowCardButton: 'button.card:has-text("Renew Now")',
+};
+
+const dynamicSelectors = {
+	purchaseTitle: ( title: string ) => `.manage-purchase__title:has-text("${ title }")`,
+};
```

Comment:
This test plan required selectors that might need to take in different text to include as part of the selector. I decided to opt for splitting the selectors at the top of the file into those that are static vs those that are dynamic so that people don't accidentally include a pointer to a function (in the case of the dynamic selectors) as a selector in a test (instead of invoking that function to get the selector).

Reply:
Interesting, I have not seen this before, but definitely useful in other situations I've faced.

I asked a question originally about 'can we keep the unified `const selectors` instead of splitting', but you've touched on why it should be split.

Reply:
That makes sense, I updated it to be in the same object

Reply:
Nice!